nephio-ansible-install/roles/clab/tasks/main.yaml

```diff
@@ -6,3 +6,60 @@
 - name: installing clab
   become: true
   shell: bash -c "$(wget -qO - https://get.containerlab.dev)"
+  when: validate_certs
+
+
+- name: Create clab installation directory
```

Comment:
L12-L65 can be put in a single block.
refer https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_blocks.html

Reply:
Good point. Done